draft-ietf-scitt-architecture.md

```diff
@@ -547,18 +547,17 @@ It indicates the Signed Statement is securing a JSON content type, and identifyi
 
 To register a Signed Statement, the Transparency Service performs the following steps:
 
-1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements on behalf of one or more Issuers.
-Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
-1. **Issuer Verification:** The Transparency Service MUST syntactically validate the Issuer's identity Claims, which may be different than the Client identity.
+1. **Client authentication:** A Client authenticates with the Transparency Service before registering Signed Statements on behalf of one or more Issuers.
+Authentication and authorization are implementation-specific and out of scope of the SCITT architecture.
+1. **Issuer Verification:** The Transparency Service MUST perform signature verification, as defined in [RFC 9052 4.4](https://datatracker.ietf.org/doc/html/rfc9052#section-4.4), and MAY perform additional checks as part of its Registration Policy.
 1. **Signature verification:** The Transparency Service MUST verify the signature of the Signed Statement, as described in {{RFC9360}}, using the signature algorithm and verification key of the Issuer.
-1. **Signed Statement validation:** The Transparency Service MUST check that the Signed Statement includes the required protected headers listed above.
-The Transparency Service MAY verify the Statement payload format, content and other optional properties.
+1. **Signed Statement validation:** The Transparency Service MUST check that the Signed Statement includes the required protected headers.
+The Transparency Service MAY validate the Signed Statement payload in order to enforce domain specific registration policies that apply to specific content types.
```

Comment:
DNS domain? Domain as in expertise?

Reply:
BCG supports the proposed changes at 555

Reply:
Apologies, some how my comment (earlier got lost)

Strictly speaking Line 555 is not necessary.

if at all one wishes to introduce this, I suggest it on the following lines.

Certain Transparency Services MAY choose to validate the Signed Statement payload for certain content types, based on their custom policies. However such validation is out of scope of this Architecture.

Reply:
@yogeshbdeshpande, the PR refers to content, meaning the content of a content-type, for instance, the contenttype could be applicaiton/json, which isn't enough. The current PR leaves this more flexible, and is shorter.

Reply:
@yogeshbdeshpande, on the editors call, we decided to move this PR forward. Please open a PR with some additional suggestions if needed.

Reply:
Would rather keep the current proposal as less is more in this case.